templates/klage-dittnav-pdfgen/klage.hbs

```diff
@@ -220,17 +221,17 @@
             </div>
 
             {{#if oversiktVedlegg}}
+            {{#unless sendesIPosten}}
```

Comment:
Denne var jo litt spennende.

Reply:
Jepp, `if not` funket ikke, artige greier.